packages/field-plugin/src/createFieldPlugin/createPluginActions/callbackQueue.ts

```diff
@@ -19,7 +19,7 @@ type CallbackMap = {
   stateChanged: Record<CallbackId, OnMessage<StateChangedMessage>>
   loaded: Record<CallbackId, OnMessage<LoadedMessage>>
   promptAI: Record<CallbackId, OnMessage<PromptAIResponseMessage>>
-  previewDimension: Record<CallbackId, OnMessage<PreviewDimensionChangeMessage>>
+  previewDimension: Record<CallbackId, OnMessage<PreviewDimensionResponse>>
```

Comment:
I'm thinking that this feels off 🤔 Since all of the other callbacks are using the same pattern with the message word on it, we should stick to this naming convention, isn't it?